pkg/minikube/assets/addons.go

```diff
@@ -403,10 +403,12 @@ func GenerateTemplateData(cfg config.KubernetesConfig) interface{} {
 		Arch            string
 		ExoticArch      string
 		ImageRepository string
+		GvisorImageRepo string
 	}{
 		Arch:            a,
 		ExoticArch:      ea,
 		ImageRepository: cfg.ImageRepository,
+		GvisorImageRepo: os.Getenv(constants.MinikubeGvisorAddonImageRepo),
```

Comment:
Would this not require normal addon users to set this environment variable? Seems like that would require an update to the docs. I'd prefer if it defaulted to cfg.ImageRepository

Reply:
Since the [default](https://github.com/kubernetes/minikube/pull/4717/files#diff-751198fe5c4c7b092069faf65234f22aR27) value is `gcr.io/k8s-skaffold`, users should never have to set the environment variable, since it should be empty. This way, they should always be using the latest version of the gvisor image.

I opted out of using cfg.ImageRepository because that repo would be applied to other images minikube pulls as well (storage provisioner, etc), which wouldn't exist. 

This way, if the env variable is set (which should only happen in integration tests), then a non-default gvisor image will be used.

Reply:
I was thinking some folks would want to override the image repo globally. My thought was to use GvisorImageRepo if it is set, but use ImageRepository if it's not set (i.e. go back to current behavior)